packages/gguf/src/gguf.ts

```diff
@@ -293,5 +295,41 @@ export async function gguf(
 		});
 	}
 
-	return { metadata, tensorInfos };
+	let paramCount: number | undefined = undefined;
+
+	if (params?.calculateParamCount) {
+		paramCount = tensorInfos
+			.map(({ shape }) => shape.reduce((acc, val) => acc * Number(val), 1))
+			.reduce((acc, val) => acc + val, 0);
```

Comment:
i agree with this part, but shouldn't we return a `Record<Dtype, number>` like in safetensors? (no strong opinion, I'm ok with only returning a number, that's fine)